libminifi/include/core/repository/AtomicRepoEntries.h

```diff
@@ -76,8 +78,7 @@ noexcept      : key_(std::move(other.key_)),
       comparator_(std::move(other.comparator_)) {
       }
 
-      ~RepoValue()
-      {
+      ~RepoValue() {
       }
```

Comment:
Correct though not related to correcting cpplint recommendations. Would you like me to run: [clang-tidy modernize-use-equals-default](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-equals-default.html)?

Reply:
You're right, I didn't notice that this was not related to the goal of the PR. Your call whether you fix it and if so, whether it will be part of this PR or separate.